samcli/lib/pipeline/bootstrap/environment_resources.yaml

```diff
@@ -67,6 +67,12 @@ Resources:
       Status: Active
       UserName: !Ref PipelineUser
 
+  PipelineUserSecretKey:
+    Type: AWS::SecretsManager::Secret
+    Condition: MissingPipelineUser
+    Properties:
+      SecretString: !Sub '{"aws_access_key_id": "${PipelineUserAccessKey}", "aws_secret_access_key": "${PipelineUserAccessKey.SecretAccessKey}"}'
+
```

Comment:
This works but I remember when we discussed this, we were going to make secret key only show up once. Saving to ssm will make the key available in AWS console forever